templates/AddSameRegionBucketDownloadRestriction.yaml

```diff
@@ -27,12 +27,6 @@ Parameters:
     AllowedPattern: '^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$'
     ConstraintDescription: 'Must be an acceptable email address syntax(i.e. joe.smith@sagebase.org)'
     Default: 'it@sagebase.org'
-  # This lambda, by default, restricts access to buckets when Gateway VPC Endpoints are enabled on an AWS account
-  # This parameter overrides that by explicitly granting bucket access to resources in this list of VPCs.
-  SameRegionResourceAccessToBucketGrantVpcAccess:
```

Comment:
there are tests that reference this, you'll need to remove from here: https://github.com/Sage-Bionetworks/aws-infra/blob/master/ci/AddSameRegionBucketDownloadRestriction-input.json#L7

Need to remove dependency here: https://github.com/Sage-Bionetworks/scicomp-provisioner/blob/master/config/prod/AMP-Mayo-Sinai-Bucket.yaml#L28

Reply:
Removed the stale test param. Is there a way to see/run the taskcat tests?

The dependency is removed in the linked PR: https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/317

Reply:
unfortunately taskcat only runs on merge because it needs AWS secrets to run.